unity-environment/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DAgent.cs

```diff
@@ -19,15 +19,16 @@ public override void CollectObservations()
         AddVectorObs(gameObject.transform.rotation.x);
         AddVectorObs(ball.transform.position - gameObject.transform.position);
         AddVectorObs(ballRb.velocity);
+        AddVectorObs(ballRb.angularVelocity);
```

Comment:
Please move all 3DBall changes to its own PR. This PR ought to be limited to Walker environment changes.